Gruntfile.js

```diff
@@ -124,7 +124,7 @@ module.exports = function(grunt) {
         tasks: ['babel']
       },
       assets: {
-        files: ['src/**', '!src/**/*.jade', '!src/**/*.stss', '!src/**/*.js'],
+        files: ['src/**/*.xml', 'src/**/*.tss', 'src/assets/**'],
```

Comment:
This is what I have done in my local projects (I haven't yet pushed it to the repository).

~~~
files: ['src/**/*.!(jade|stss|js)'],
~~~